channels/rdpecam/client/camera.h

```diff
@@ -124,11 +117,16 @@ typedef struct
 
 static INLINE CAM_MEDIA_FORMAT streamInputFormat(CameraDeviceStream* stream)
 {
-	return stream->formats.inputFormat;
+	return stream->inputFormat;
 }
 static INLINE CAM_MEDIA_FORMAT streamOutputFormat(CameraDeviceStream* stream)
 {
-	return stream->formats.outputFormat;
+#if !defined(WITH_FORMAT_PASSTHROUGH)
+	return CAM_MEDIA_FORMAT_H264;
+#else
+	return (stream->inputFormat == CAM_MEDIA_FORMAT_MJPG_H264) ? CAM_MEDIA_FORMAT_H264
```

Comment:
did miss this in the last pr, `CAM_MEDIA_FORMAT_MJPG_H264` is kind of a misuse of the `enum`.
no other (clean) way to use this? (e.g. split input & output format?)